.github/workflows/add_redirects_label.yml

```diff
@@ -0,0 +1,42 @@
+name: Add redirects-needed label
```

Comment:
Just curious, did you test the workflow somewhere?

Reply:
Yea I've been testing it out in my test org with a `docs` fork here: https://github.com/timngyn-test-org/docs/pull/25 (there are more PRs too, but I've closed them)